api/lib/swaggers.js

```diff
@@ -19,6 +19,15 @@ const swaggerOptionsLivretScolaire = {
   jsonPath: '/swagger.json',
 };
 
+const swaggerOptionsPoleEmploi = {
+  routeTag: 'pole-emploi',
+  info: {
+    'title': 'Pix Pôle emploi open api',
```

Comment:
**suggestion** : les autres titres sont de la forme "Welcome to ..." pourquoi ne pas faire pareil ?

Reply:
Gravitee attend un titre "court" (pour une meilleure lisibilité), c'est pour ça que nous avons enlevé le "Welcome to" qui n'apporte rien. (Vu avec Youness)